### ISSUE TITLE
Copied files from EXT4 drive to another EXT4 don't retain executable attribute.

### ISSUE THREAD

**Issue by Harry-W-Haines-III** [pop-os/cosmic-files#346]
Just noticed copying some bash scripts from a portable USB SSD formatted in EXT4 to /home/Downloads didn't keep the +x and had to manually reset them +x on the destination folder.  I have not seen this elsewhere.

**Comment by iamkartiknayak** [comment]
Not only `ext4`, it behaves the same for `NTFS` too.

**Comment by jackpot51** [comment]
Is this still the case?

**Comment by Harry-W-Haines-III** [comment]
Yes, it is.  I just updated everything, rebooted and checked again from one NVME volume formatted in ext4 to another.

**Comment by jackpot51** [comment]
I cannot reproduce this, can you provide the `stat` command output on the file before and after copying?

**Comment by Harry-W-Haines-III** [comment]
[stats.txt](https://github.com/user-attachments/files/18447373/stats.txt)

Issue looks to be resolved.

**Comment by iamkartiknayak** [comment]
The issue seems to have been resolved!

**Comment by jackpot51** [comment]
Thanks for checking!
